apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -173,7 +173,13 @@ class TopInstructionsCSP extends Component {
           method: 'GET',
           contentType: 'application/json;charset=UTF-8'
         }).done(data => {
-          this.setState({feedbacks: data});
+          // If student has feedback make their default tab the feedback tab instead of instructions
+          if (
+            data[0] &&
+            (data[0].comment !== '' || data[0].performance !== null)
```

Comment:
nit: use the length of the `comment` string rather than checking against an empty string:

```suggestion
            (data[0].comment.length === 0 || data[0].performance !== null)
```